share/doc/revng/user-manual/tutorial/running-analyses.md

```diff
@@ -25,13 +25,15 @@ $ grep -A7 'Segments:' example.yml
 Segments:
   - StartAddress:    "0x400000:Generic64"
     VirtualSize:     2528
-    StartOffset:     0
     FileSize:        2528
     IsReadable:      true
-    IsWriteable:     false
     IsExecutable:    true
+    Relocations:
+      - Address:         "0x401d88:Generic64"
 ```
 
+**ivan**: no clue where `Relocations` came from. Do you have an idea? Or should I go looking for it blindly?
```

Comment:
I don't understand your question?

I guess the input binary has a relocation that was not popping up before, for some reason?